ApollosProject/apollos-apps
# fix(gratitude): update enter button text in keyboard to done

## ✏️ Description
- update enter button text in keyboard to `done` instead of `return`


## 🔬 To Test
1. Navigate to the Daily tab
2. Open up the Gratitude Experience
3. Navigate to the screen that allows you to input a Gratitude
4. When the keyboard is displayed, look at the "enter" or "return" key
5. Text should read `done`

## 📸 Screenshots

![image](https://user-images.githubusercontent.com/7557597/205728235-d627dac9-3986-4b9b-9b6d-8932a774693f.png)


## 📝 Checklist
